swagger.yaml

```diff
@@ -236,20 +236,11 @@ definitions:
         type: string
       sourceType:
         type: string
-        description: > 
+        description: >
           Type of data source being accessed directly. Possible values are "api", "database", or "other".
       otherSourceType:
         type: string
         description: "Explanation if sourceType is 'other'"
-      database:
```

Comment:
Hmm good point. I added a field to handle short source descriptions, like "ODS" or "Banner"